backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceAsync.java

```diff
@@ -116,30 +116,35 @@ public CompletableFuture<Set<Person>> savePatients(byte[] content, UUID facility
         Person newPatient =
-            new Person(
-                currentOrganization,
-                assignedFacility.orElse(null),
-                null, // lookupid
-                extractedData.getFirstName().getValue(),
-                extractedData.getMiddleName().getValue(),
-                extractedData.getLastName().getValue(),
-                extractedData.getSuffix().getValue(),
-                parseUserShortDate(extractedData.getDateOfBirth().getValue()),
-                address,
-                country,
-                parsePersonRole(extractedData.getRole().getValue(), false),
-                extractedData.getEmail().getValue() == null
-                    ? Collections.emptyList()
-                    : List.of(extractedData.getEmail().getValue()),
-                convertRaceToDatabaseValue(extractedData.getRace().getValue()),
-                convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()),
-                null, // tribalAffiliation
-                convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()),
-                parseYesNoUnk(extractedData.getResidentCongregateSetting().getValue()),
-                parseYesNoUnk(extractedData.getEmployedInHealthcare().getValue()),
-                null, // preferredLanguage
-                null // testResultDeliveryPreference
-                );
+            Person.builder()
+                .organization(currentOrganization)
+                .facility(assignedFacility.orElse(null))
+                .birthDate(parseUserShortDate(extractedData.getDateOfBirth().getValue()))
+                .address(address)
+                .country(country)
+                .role(parsePersonRole(extractedData.getRole().getValue(), false))
+                .emails(
+                    extractedData.getEmail().getValue() == null
+                        ? Collections.emptyList()
+                        : List.of(extractedData.getEmail().getValue()))
+                .race(convertRaceToDatabaseValue(extractedData.getRace().getValue()))
+                .ethnicity(convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()))
+                .gender(convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()))
+                .genderIdentity(extractedData.getGenderIdentity().getValue())
+                .residentCongregateSetting(
+                    parseYesNoUnk(extractedData.getResidentCongregateSetting().getValue()))
+                .employedInHealthcare(
+                    parseYesNoUnk(extractedData.getEmployedInHealthcare().getValue()))
+                .firstName(extractedData.getFirstName().getValue())
+                .middleName(extractedData.getMiddleName().getValue())
+                .lastName(extractedData.getLastName().getValue())
+                .suffix(extractedData.getSuffix().getValue())
+                .notes(extractedData.getNotes().getValue())
```

Comment:
Right now since we're not doing any string interpolation this column can either get filled with `null` (if the columns aren't specified) or the empty string if the column is specified but there isn't any value for that row. I left it this way since we're doing the same thing in other places.

lmk if we're not ok with this and I can go back and clean it up here / in other settings